src/components/Icons/CollapseOpenIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function CollapseOpenIcon() {
+    return (
+        <svg width="10" height="6" viewBox="0 0 10 6" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path d="M9 5L5 1L1 5" stroke="#2F353F" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+        </svg>
+    )
+}
+
+export default CollapseOpenIcon;
```

Comment:
missing line

Reply:
not sure what "missing line" means. Are you saying missing "endline"? 

we need to decide about formatter. I'm using `yarn lint --fix` to determine formatting. We need to agree about this stuff.